### PR TITLE
HomeApp: add Web clips based on data from array

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**

### CODE REVIEW

File: examples/mobile/HomeApp/js/app.js [hunk 1]
```diff
@@ -9,7 +9,12 @@
 		},
 		// TODO: remove below directive after webClipList use
 		// eslint-disable-next-line no-unused-vars
-		webClipList = [];
+		webClipList = [],
```
Comment (lslachciak) [inline examples/mobile/HomeApp/js/app.js]: webClipList should by default contain at least this what is defined in activeWebClips

webClipList  >= activeWebclips

Comment (lslachciak) [reply examples/mobile/HomeApp/js/app.js]: activeWebclips --> activeWeb**C**lips

File: examples/mobile/HomeApp/js/app.js [hunk 2]
```diff
@@ -9,7 +9,19 @@
 		},
 		// TODO: remove below directive after webClipList use
 		// eslint-disable-next-line no-unused-vars
-		webClipList = [];
+		webClipList = [
```
Comment (singa2000) [inline examples/mobile/HomeApp/js/app.js:12]: I have a question.
It looks you pre-defined the webClipList.
If I remove webclip (example "rv-remote-control"), is it also update??

**Comment by lslachciak** [comment]
I rebased PR basing on the latest repo

File: examples/mobile/HomeApp/js/app.js (hunk 2 above)
Comment (lslachciak) [reply examples/mobile/HomeApp/js/app.js:12]: webClipList will be updated on application start -
loadWebClipList() updates it basing on the api/webcliplist

**Review by lslachciak — APPROVED** [review]

**Review by lslachciak — COMMENTED** [review]
I'm testing this PR now

**Review by lslachciak — APPROVED** [review]

**Comment by lslachciak** [comment]
I'll make improvements in separate PR

**Pull request merged**